lib/modules/manager/cargo/extract.ts

```diff
@@ -60,6 +62,9 @@ function extractFromSection(
       } else if (git) {
         currentValue = '';
         skipReason = 'git-dependency';
+      } else if (workspace) {
+        currentValue = '';
+        skipReason = 'local-dependency';
```

Comment:
Should we distinguish this from path dependencies by saying `'inherited-dependency'` or `'workspace-dependency'`?

Reply:
inherited-dependency seems like a good fit.